src/icon-manifest/manifestMerger.ts

```diff
@@ -87,7 +87,7 @@ export function toggleAngularPreset(
   disable: boolean,
   files: models.IFileCollection): models.IFileCollection {
   const icons = files.supported
-    .filter(x => x.icon.startsWith('ng_') && (x.icon.endsWith('_js') || x.icon.endsWith('_ts')))
+    .filter(x => x.icon.startsWith('ng_') && !x.icon.endsWith('2'))
```

Comment:
Checking against numbers and not a `string` representation of numbers doesn't violate KISS and already provides the right condition for expansion.